code/modules/food_and_drinks/machinery/oven.dm

```diff
@@ -101,13 +101,22 @@
 
 
 /obj/machinery/oven/attackby(obj/item/I, mob/user, params)
-	if(open && !used_tray && istype(I, /obj/item/plate/oven_tray))
+	//if(open && used_tray && I.atom_storage)
+		//Stop containers from beating up our oven in general.
+		//return
+
+	if(open && !used_tray && I.atom_storage)
```

Comment:
what happened here? why is this code commented out? can this just be early returned?

also, please, give `I` a better name, like `weapon` or something

Reply:
Woops, forgot to delete that comment. I was the original name, but I'll change it, magic numbers bad.